src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -9,12 +12,16 @@ public class ShopService {
     /*** This class is used for creating new e-shop object and initializing it with some products.
      */
 
+
     public Eshop createShop() {
         /** Should create new e-shop object and initialize it with some products.
          * Should return e-shop object.
          */
-       ArrayList<Product> products = createProducts();
-        throw new UnsupportedOperationException("createShop() is not implemented yet");
+
+
+        ArrayList<Product> products = createProducts();
+        Eshop eshop = new Eshop("name", products);
+        return eshop;
```

Comment:
You could inline this and save couple of lines of code like
`return new Eshop("name", createProducts());`